exporter/awsemfexporter/config.go

```diff
@@ -9,8 +9,8 @@ import (
 	"go.opentelemetry.io/collector/component"
 	"go.uber.org/zap"
 
-	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/awsutil"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cwlogs"
+	"github.com/amazon-contributing/opentelemetry-collector-contrib/internal/aws/awsutil"
+	"github.com/amazon-contributing/opentelemetry-collector-contrib/internal/aws/cwlogs"
```

Comment:
Q: aren't XRayExporter and EMF exporter are already go modules now that used by CWAgent? And it doesn't need to change these dependencies, why do we need to do it in this branch?
https://github.com/amazon-contributing/opentelemetry-collector-contrib/blob/aws-cwa-dev/exporter/awsemfexporter/datapoint.go#L17

Reply:
CWAgent does use these as go modules from the `aws-cwa-dev` branch, but it has to do it via:
1. `require 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxrayexporter v0.103.0`
2. `replace 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxrayexporter => github.com/amazon-contributing/opentelemetry-collector-contrib/exporter/awsxrayexporter v0.0.0-20241024160840-0c7076070f12`



https://github.com/aws/amazon-cloudwatch-agent/blob/main/go.mod#L11-L12

So today it doesn't actually use a module from "`amazon-contributing`", but it replaces the `open-telemetry` exporter modules with the `amazon-contributing` modules.

-------------

The changes in this PR makes it possible to just `require github.com/amazon-contributing/opentelemetry-collector-contrib/exporter/awsxrayexporter v0.103.0`